mirrord/agent/src/file.rs

```diff
@@ -168,6 +173,16 @@ impl FileManager {
                 let xstat_result = self.xstat(path, fd, follow_symlink);
                 Ok(FileResponse::Xstat(xstat_result))
             }
+
+            FileRequest::FdOpenDir(FdOpenDirRequest { remote_fd }) => {
+                let open_dir_result = self.fdopen_dir(remote_fd);
+                Ok(FileResponse::OpenDir(open_dir_result))
```

Comment:
Not affected by this PR, but would it be good style to move out the `Ok` to have a single `Ok` around the whole `match` instead of wrapping the return value of every single arm with `Ok`? - If yes, we could take this opportunity to do this little change.

Reply:
Done